src/lib/components/cookies/utils/Cookie.js

```diff
@@ -47,29 +42,27 @@ export const setCookies = (tag) => {
 export const deleteCookies = (tag) => {
     Cookies.remove("_ga");
     Cookies.remove("_gid");
+    Cookies.remove(`_ga_${tag?.slice(2)}`)
     Cookies.remove(`_gat_gtag_${tag}`);
 
     window[`ga-disable-${tag}`] = true;
-
-
-    console.info("Removed cookies and disabled tracking.");
 };
 
 
 export const handleCookieAccept = (accepted, tag) => {
     const
         today = new Date(),
         [year, month, day] = [today.getFullYear(), today.getMonth(), today.getDate()],
-        cookieExpiryDate = new Date(year, month + 1, day).toUTCString();
-
+        cookieExpiryDate = new Date(year + 1, month, day).toUTCString();
     if (accepted) {
-        document.cookie = `cookies_policy_21_3=${encodeURIComponent('{"essential":true,"usage":true,"preferences":true}')}; expires=${cookieExpiryDate};`;
+        document.cookie = `cookies_policy=${encodeURIComponent('{"essential":true,"usage":true,"preferences":true}')}; expires=${cookieExpiryDate};`;
```

Comment:
Just thought this is a bad idea  it doesn't look like your keeping existing cookies. This feels like it would break cookie authentication